core/src/main/kotlin/org/ghrobotics/lib/mathematics/twodim/trajectory/constraints/DifferentailDriveDynamicsConstraint.kt

```diff
@@ -0,0 +1,109 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * Copyright 2019, Green Hope Falcons
+ */
+
+package org.ghrobotics.lib.mathematics.twodim.trajectory.constraints
+
+import edu.wpi.first.wpilibj.geometry.Pose2d
+import edu.wpi.first.wpilibj.kinematics.ChassisSpeeds
+import edu.wpi.first.wpilibj.kinematics.DifferentialDriveKinematics
+import edu.wpi.first.wpilibj.kinematics.DifferentialDriveWheelSpeeds
+import edu.wpi.first.wpilibj.trajectory.constraint.TrajectoryConstraint
+import org.ejml.simple.SimpleMatrix
+import org.ghrobotics.lib.mathematics.epsilonEquals
+import org.ghrobotics.lib.mathematics.kEpsilon
+import org.ghrobotics.lib.mathematics.units.Meter
+import org.ghrobotics.lib.mathematics.units.SIUnit
+import org.ghrobotics.lib.mathematics.units.derived.Volt
+import org.ghrobotics.lib.mathematics.units.inMeters
+import org.ghrobotics.lib.physics.MotorCharacterization
+import kotlin.math.max
+import kotlin.math.min
+import kotlin.math.sign
+import kotlin.math.abs
+
+@Suppress("unused")
+class DifferentailDriveDynamicsConstraint(
```

Comment:
Should be `DifferentialDriveDynamicsConstraint`

Reply:
Resolved in 14a4432aa05127712f73e05f50bd8a1a2448ae45